### DISCUSSION TITLE
Short DuckDB videos from @mneedham

### DISCUSSION THREAD

**Discussion by jexp** [duckdb/duckdb#6237] [Show and tell]
My buddy Mark has created a bunch of really helpful, short DuckDB videos covering a number of topics

https://www.youtube.com/watch?v=fZj6kTwXN1U&list=PLw2SS5iImhEThtiGNPiNenOr2tVvLj6H7

- getting started
- reading parquet files
- exploring nested JSON files and turning them into tables to query
- rendering cli outputs with youplot
- exploring and analyzing paquet schema with SQL
- loading csv files obvs
- data quality in CSV and parquet
- row groups in parquet

**Comment by derekmahar** [comment]
Great videos! I like Mark's bite-sized tutorial format.  I hope he produces many more videos like these!
